src/components/MerchantsPage/gam/MegaGamListItem/MegaGamListItem.tsx

```diff
@@ -72,30 +81,79 @@ const MegaGamListItem = ({ campaign }: Props) => {
 };
 
 const Container = styled.div`
-  background-color: #f7f7f7;
+  background-color: ${({ isActive }: { isActive: boolean }) =>
+    isActive ? '#f7f7f7' : '#ffffff'};
   display: flex;
   flex-direction: column;
-  margin-bottom: 56px;
+  margin-bottom: ${({ isActive }: { isActive: boolean }) =>
+    isActive ? '56px' : '0'};
 `;
 
-// TODO: Figure out how this image should scale.
-const HeroImage = styled.div`
-  background-image: url('${(props: { heroImageUrl: string }) =>
-    props.heroImageUrl}');
-  background-size: cover;
-  height: 252px;
-  width: 100%;
+const ImageContentContainer = styled.div`
+  display: flex;
+  flex-direction: ${({ isActive }: { isActive: boolean }) =>
+    isActive ? 'column' : 'row'};
+`;
```

Comment:
all these inactive checks seem unindeal, but I can't think of another way of doing it

Reply:
yeah it seems product/design is set on the separate layouts for active vs inactive